src/Goerli-DssSpell.sol

```diff
@@ -86,278 +27,107 @@ contract DssSpellAction is DssAction {
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
     string public constant override description = "Goerli Spell";
 
-    address constant MCD_VEST_DAI = 0x59B1a603cAC9e38EA2AC2C479FFE42Ce48123Fd4;
-    address constant MCD_VEST_MKR = 0x183bE7a75B8b5F35236270b060e95C65D82f5fF9;
-
-    uint256 constant TWO_PCT            = 1000000000627937192491029810;
-    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
-    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
-    uint256 constant FIVE_PCT           = 1000000001547125957863212448;
-    uint256 constant SIX_PCT            = 1000000001847694957439350562;
-    uint256 constant SEVEN_PCT          = 1000000002145441671308778766;
-
-    uint256 constant MILLION = 10 ** 6;
-    uint256 constant WAD     = 10 ** 18;
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //
+    uint256 constant ONE_PCT_RATE = 1000000000315522921573372069;
+
+    address constant LERP_FAB = 0xbBD821c291c492c40Db2577D9b6E5B1bdAEBD207;
```

Comment:
nit: missing License on Etherscan (in Other Settings)
to keep in mind for future deployments ;)

Reply:
I don’t really know how to fix that, please let me know if you do, thank you

Reply:
not sure could be fixed after being verified on etherscan, just a note for mainnet deployment.

Reply:
I used the `verify.py` script from this répository. But looking at the Etherscan API documentation, it only supports license numbers from 1 to 12... ours in number 13... https://docs.etherscan.io/tutorials/verifying-contracts-programmatically

Reply:
oh! yeah, I reported this issue to @e18r, we should fix that.

Reply:
We should use 12 (GPLv3) till that's fixed